### ISSUE TITLE
Actionable notifications only show up when app is in foreground

### ISSUE THREAD

**Issue by dshokouhi** [home-assistant/home-assistant-android#337]
**Home Assistant Android version:**
1.5.1 (101)
**Android version:**
10
**Phone model:**
Pixel 4 XL
**Home Assistant version:**
0.104.3
**Last working Home Assistant release (if known):**
n/a
**Description of problem:**
The action in the actionable notification does not show up when the user is on the home screen or in any other app.  The rest of the notification shows up fine.  If I open the app and trigger the same service call the action shows up as expected.

Steps to reproduce:
1. Open app, send below service data and observe notification is as expected.
2. Go to home screen, send below service data again.  Observe the action button is missing.
3. Open app again and send below service data.  Observe notification is as expected.

Service call:

```
  - service: notify.mobile_app_pixel_4_xl
    data:
      message: test
      title: title
      data:
        data:
          action_1_key: KEY_ONE
          action_1_title: BUTTON
```
**Traceback (if applicable):**

```

```

**Additional information:**
![image](https://user-images.githubusercontent.com/1634145/73630401-c2cc2f00-460a-11ea-9266-970a8c8df320.png)

![image](https://user-images.githubusercontent.com/1634145/73630410-c8c21000-460a-11ea-99bf-1a368d0daa00.png)

**Comment by quthla** [comment]
I can confirm this issue. If seems the system creates the notification and does not let the app handle it for some reason.

**Comment by JBassett** [comment]
Yep, we'll need to move the title and message into the data object on the fcm server side and extract them on device.

**Comment by acordill** [comment]
Fixed in 1.5.1 (103)+
Should be closed.
